src/Appender/RollingFileAppender.cs

```diff
@@ -1283,6 +1287,21 @@ protected void RollFile(string fromFile, string toFile)
 					using(SecurityContext.Impersonate(this))
 					{
 						System.IO.File.Move(fromFile, toFile);
+					    if ( FileExists( fromZipFileName ) )
+					    {
+					        System.IO.File.Move( fromZipFileName, toZipFileName );
+					    }
+					    else
+					    {
+					        using ( System.IO.Compression.ZipArchive archive =
+					            System.IO.Compression.ZipFile.Open( toZipFileName, ZipArchiveMode.Create ) )
+					        {
+					            String archivePath = ( $"{System.IO.Path.GetDirectoryName( toFile )}" );
+					            //archive.CreateEntryFromFile( toFile, toZipFileName );
+					            archive.CreateEntryFromFile( toFile, toZipFileName, CompressionLevel.Fastest );
+					            LogLog.Debug( declaringType, $"Archiving {toFile} to {toZipFileName}." );
```

Comment:
Minor nit, can you move the LogLog statement to before the archiving action?

Reply:
Also I was wondering if we should use the CompressionLevel.Optimal instead of Fastest here. Any thoughts?